Gems/Atom/Feature/Common/Assets/Materials/Pipelines/MainPipeline/ForwardPass_SkinLighting.azsli

```diff
@@ -16,6 +16,8 @@
 #define ENABLE_ANISOTROPY 0
 #define ENABLE_SHADER_DEBUGGING 1
 #define ENABLE_CLEAR_COAT 0
+#define ENABLE_DUAL_SPECULAR 1
+#define FORCE_IBL_IN_FORWARD_PASS 1
```

Comment:
Do we want this for main pipeline? I thought this was enabled for LowEnd pipeline